mits/models.py

```diff
@@ -0,0 +1,179 @@
+from mits import *
+
+
+@Session.model_mixin
+class SessionMixin:
+    def logged_in_mits_team(self):
+        try:
+            return self.mits_team(cherrypy.session['mits_team_id'])
+        except:
+            raise HTTPRedirect('../mits_applications/team')
+
+    def mits_teams(self):
+        return (self.query(MITSTeam)
+                    .options(joinedload(MITSTeam.applicants),
+                             joinedload(MITSTeam.games),
+                             joinedload(MITSTeam.schedule),
+                             joinedload(MITSTeam.pictures))
+                    .order_by(MITSTeam.name))
+
+    def delete_mits_picture(self, picture):
+        self.delete(picture)
+        try:
+            os.remove(picture.filepath)
+        except:
+            log.error('unexpected error deleting MITS image {}', picture.filepath)
+        self.commit()
```

Comment:
Is it a problem that we commit whether or not the operation succeeded..?

Reply:
Good question - I've just added the following comment explaining why it still makes sense to commit in the case of an exception:

```
# Regardless of whether removing the file from the filesystem succeeded,
# we still want the delete it from the database.  The most likely cause
# of failure is if the file was already deleted or is otherwise not
# present, so it wouldn't make sense to keep the database record around.
```